src/main/java/eu/dzhw/fdz/metadatamanagement/variablemanagement/service/VariableManagementService.java

```diff
@@ -47,9 +76,20 @@ public class VariableManagementService implements CrudService<Variable> {
 
   private final VariableCrudHelper crudHelper;
 
+  private final ObjectMapper objectMapper = new ObjectMapper();
+
+  private final RestHighLevelClient client;
+
+  private final Gson gson;
```

Comment:
We should consolidate the usage of JSON libraries. Since GSON was problematic in the past regarding dependencies and JDK compatibility we should try and replace all calls with Jackson ObjectMapper functionality.

Reply:
As switching to Jackson ObjectMapper lead to a lot of changes in various types of document classes we decided to stay with gson in this case.